Library/Homebrew/cmd/update.sh

```diff
@@ -781,68 +781,73 @@ EOS
 
   if [[ -z "${HOMEBREW_NO_INSTALL_FROM_API}" ]]
   then
-    mkdir -p "${HOMEBREW_CACHE}/api"
+    local api_cache="${HOMEBREW_CACHE}/api"
+    mkdir -p "${api_cache}"
 
-    for formula_or_cask in formula cask
+    for json in formula.jws cask.jws homebrew_core_tap_migrations homebrew_cask_tap_migrations
```

Comment:
This really should be JWS too for security reasons.

Not overly difficult to do but if we do go this route we really shouldn't skip doing so.

Reply:
Done!